lib/liveValidation/operationValidator.ts

```diff
@@ -276,17 +299,32 @@ export const schemaValidateIssueToLiveValidationIssue = (
 
     let skipIssue = false;
     issue.pathsInPayload = issue.jsonPathsInPayload.map((path, idx) => {
-      const isMissingRequiredProperty = issue.code === "OBJECT_MISSING_REQUIRED_PROPERTY";
-      const isBodyIssue = path.startsWith(".body");
-
       if (issue.code === "MISSING_RESOURCE_ID") {
         // ignore this error for sub level resources
         if (path.includes("properties")) {
           skipIssue = true;
           return "";
         }
+      } else if (issue.code === "INVALID_TYPE" && isArmCall !== undefined && isArmCall === false) {
+        if (issue.schemaPath.includes("additionalProperties")) {
```

Comment:
what is the rule for additionalProperties, why we can ignore the invalid_type for additionalProperties?

Reply:
This work is what we discussed in last two sync up meeting. The context is: oav new version added the `additionalProperties` type validation which appears to be a breaking change to service team. There's no easy approach to identify the service who has inconsistence between annotations of `additionalProperties` and API implementation. 

We don't want to introduce LSI by this new version release.  That's why we have this PR. Feel free to ping me offline for further questions.